content/hi/docs/tasks/tools/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "उपकरण स्थापित करें"
+description: अपने कंप्यूटर पर कुबेरनेट्स टूल सेट करें.
+weight: 10
+no_list: true
+---
+
+## kubectl
+
+<!-- overview -->
+कुबेरनेट्स कमांड-लाइन टूल, [कुबेक्टल](/docs/reference/kubectl/kubectl/), 
+आपको कुबेरनेट्स क्लस्टर पर कमांड चलाने की अनुमति देता है।
+
+आप एप्लिकेशन को डिप्लॉय करने, क्लस्टर संसाधनों का निरीक्षण 
+और प्रबंधन करने और लॉग देखने के लिए कुबेक्टल का उपयोग कर सकते हैं।
+
+कुबेक्टल संचालन की पूरी सूची सहित अधिक जानकारी के लिए, देखें 
```

Comment:
```suggestion
kubectl संचालन की पूरी सूची सहित अधिक जानकारी के लिए, देखें 
```

Reply:
Better write kubectl as kubectl as transliterating makes it complex.
FYI - Its pronounced "Kube C T L"